README.md

```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
-```
+#### Create a cluster in an already existing infrastructure
+
+Same as the previous section, but you can skip the infrastructure creation phase
+by not filling the section `infrastructure` in the `furyctl.yaml` file and
+running `furyctl create cluster --skip-phase infrastructure`.
+
+#### Deploy a cluster step by step
+
+The cluster creation process can be split into three phases:
 
-### Deploy a cluster from an already existing infrastructure
+1. Infrastructure
+2. Kubernetes
+3. Distribution
 
-The following workflow describes a setup of a cluster using an already existing underlay infrastructure.
+The `furyctl create cluster` command will execute all the phases by default,
+but you can limit the execution to a specific phase by using the `--phase` flag.
+
+To create a cluster step by step, you can run the following command:
 
 ```bash
-+--------------------------+   +--------------------------+
-+ furyctl cluster init     +-->+ furyctl cluster apply    |
-+--------------------------+   +--------------------------+
+furyctl create cluster --phase infrastructure
 ```
 
-### Installers
+If you choose to create a VPN in the infrastructure phase, you can automatically connect to it by using the flag `--vpn-auto-connect`.
 
-To deploy all the infrastructure components `furyctl` uses *installers*.
+```bash
+furyctl create cluster --phase kubernetes
+```
 
-> You can use an environment variable to avoid passing the token via console: `FURYCTL_TOKEN`.
+After running the command, remember to export the `KUBECONFIG` environment variable to point to the generated kubeconfig file or
+to use the flag `--kubeconfig` in the following command.
 
-Contact [sales@sighup.io](mailto:sales@sighup.io) to get more details about this feature.
+```bash
+furyctl create cluster --phase distribution
+```
 
-#### Bootstrap
+##### Infrastructure
 
-The available `bootstrap` provisioners are:
+The available `infrastructure` provisioners are:
 
-| Provisioner | Description                                                                                                                                            |
-| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `aws`       | It creates a VPC with all the requirements to deploy a Kubernetes Cluster. It also includes a VPN instance easily manageable by using `furyagent`.     |
+| Provisioner | Description                                                                                                                                        |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `aws`       | It creates a VPC with all the requirements to deploy a Kubernetes Cluster. It also includes a VPN instance easily manageable by using `furyagent`. |
```

Comment:
this seems to be related to the old furyctl

Reply:
removed